docs/components/console/manage-organization/advanced-search.md

```diff
@@ -0,0 +1,26 @@
+---
+id: advanced-search
+title: Advanced search
+description: "Use the search bar to navigate through Camunda applications and locate necessary resources and documentation."
+---
+
+To ease navigation throughout Console, utilize the **search functionality** available in Camunda Platform 8 SaaS under the **Console** component.
+
+This search functionality allows users to:
+
+- Navigate between apps, actions, and tasks (e.g. go to Modeler, invite users, API clients, etc.)
+- Locate Camunda project assets (e.g. BPMN, DMN models, and clusters).
+- Find all entries in the Camunda forum, documentation, and public GitHub issues.
+
+## Open the search bar
+
+Press `ctrl`+`k`, `⌘`+`k`, or click the magnifier in the top navigation bar to open the search bar.
```

Comment:
This is non-blocking, but I assume each command is specific to operating system? If so, I would likely note those in parentheses.